config.go

```diff
@@ -95,17 +96,18 @@ func loadConfig() (*config, error) {
 		RawRPCListener:  fmt.Sprintf("localhost:%d", defaultRPCPort),
 		RawRESTListener: fmt.Sprintf("localhost:%d", defaultRESTPort),
 		Service: &service.Config{
-			N:                        defaultN,
```

Comment:
since you are removing the config N/InitialRoundDuration, please also update comments that reference these configuration. otherwise this codebase is made more confusing by this PR